ash1eygrace/ai-content
# Feature: Display Featured Generators on Homepage Dynamically

This solves [Refactor: Home.js's Generator List to Dynamically Display Data](https://github.com/ash1eygrace/ai-content/issues/26) #26  

This PR adds a new feature that allows for dynamic display of featured generators on the homepage. The changes enable easier management of featured generators by simply modifying the `featured` value in the `GeneratorList` data objects. 

**Key changes:**

1. Added a `featured` key-value pair to the generator objects in `GeneratorList`.
2. Modified the `Home` component to filter and display only the featured generators from the `GeneratorList`.

**Testing Instructions:**

1. Pull the latest changes from the branch and install any new dependencies if necessary.
2. Start the development server using `npm start`
3. Navigate to the homepage and verify that only the featured generators (as defined in `GeneratorList`) are displayed.
4. Modify the `featured` key-value pair in the `GeneratorList` data objects to test different combinations of featured generators.
5. Check homepage and ensure that the displayed featured generators match the updated `featured` values in `GeneratorList`.
